templates/news/news-en.html

```diff
@@ -3,224 +3,476 @@
+							<option>non-Canadians</option>
+							<option>non-governmental organizations</option>
+							<option>parents</option>
+							<option>persons with disabilities</option>
+							<option>rural community</option>
+							<option>scientists</option>
+							<option>seniors</option>
+							<option>students</option>
+							<option>travellers</option>
+							<option>veterans</option>
+							<option>visitors to Canada</option>
+							<option>women</option>
+							<option>youth</option>
+						</select>
 					</div>
-				</div>
-			</form>
-		</details>
-		<section class="lnkbx well">
+					<div class="form-group">
+						<label for="dt_location">Region</label>
+						<select class="form-control" id="dt_location" name="dt_location" data-column="5">
+							<option value="">Any</option>
+							<option>Atlantic</option>
+							<option>British Columbia</option>
+							<option>Ontario</option>
+							<option>Prairies</option>
+							<option>Quebec</option>
+							<option>Territories</option>
+						</select>
+					</div>
+					<div class="row">
+						<div class="col-xs-6">
+							<button type="submit" class="btn btn-primary" aria-controls="dataset-filter">Filter</button>
+						</div>
+						<div class="col-xs-6">
+							<button type="reset" class="btn btn-default">Reset filters</button>
+						</div>
+					</div>
+				</form>
+			</details>
+		</section>
+		<section>
 			<h2 class="h4">Resources</h2>
-			<ul>
-				<li><a href="#">Canada Gazette</a></li>
-				<li><a href="#">Hansard</a></li>
-				<li><a href="#">Media Contacts</a></li>
+			<ul class="list-group">
+				<li class="list-group-item"><a href="#">Canada Gazette</a></li>
+				<li class="list-group-item"><a href="#">Hansard</a></li>
+				<li class="list-group-item"><a href="#">Media Contacts</a></li>
 			</ul>
 		</section>
 	</div>
 	<div class="col-md-9">
-		<table class="wb-tables table nws-tbl" id="dataset-filter" aria-live="polite" data-wb-tables='{
-		"bDeferRender": true,
-		"ajaxSource": "https://wet-boew.github.io/wet-boew/demos/tables/ajax/datasource.json",
-		"order": [1, "desc"],
-		"columns": [
-			{ "data": "TITLE", "className": "nws-tbl-ttl h4" },
-			{ "data": "PUBDATE", "className": "nws-tbl-date" },
-			{ "data": "DEPT", "className": "nws-tbl-dept" },
-			{ "data": "TYPE", "className": "nws-tbl-type" },
-			{ "data": "TEASER",  "className": "nws-tbl-desc" },
-			{ "data": "LOCATION",  "visible": false },
-			{ "data": "AUDIENCE",  "visible": false },
-			{ "data": "SUBJECT", "visible": false },
-			{ "data": "MINISTER", "visible": false }
-		]}'>
-			<thead class="wb-inv">
-			<tr>
-				<th>Title</th>
-				<th>Publication date</th>
-				<th>Department</th>
-				<th>News Type</th>
-				<th>Summary</th>
-				<th>Location</th>
-				<th>For</th>
-				<th>Subject</th>
-				<th>Minister</th>
-			</tr>
-			</thead>
-		</table>
+		<p>To access all articles, please visit the <a href="https://www.canada.ca/en/news/advanced-news-search.html">advanced search</a>.</p>
```

Comment:
🤔 this paragraph representative heading is the first h2 "Promotions". May be we can wrap the first implicit h2 into its own section then this paragraph would be represented by the h1 of the page which is not the best but kind of better.